src/Test.UnitTests.Sarif/TestData/SarifCurrentToVersionOneVisitor/ExpectedOutputs/OneRunWithAutomationDetails.sarif

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-1.0.0.json",
+  "version": "1.0.0",
+  "runs": [
```

Comment:
>runs [](start = 3, length = 4)

i think i would have investigated introducing these patterns into some other test file rather than creating a detail test file for these fixes. just a gut call.